src/com/company/redi_exercises/Main.java

```diff
@@ -0,0 +1,186 @@
+        ArrayList<String> inputWords = new ArrayList<>();
+        for (int i = 0; i <= numberWords; i++) {
+            inputWords.add(scanner.nextLine());
+        }
+        System.out.println("Min length: ");
+        int numberLetters = scanner.nextInt();
+        System.out.println(removeShortWords(inputWords, numberLetters));
+    }
+
+    public static ArrayList<String> removeShortWords(ArrayList<String> words, int minLength) {
+        ArrayList<String> earnedWords = new ArrayList<>();
+        for (String word : words) {
+            if (word.length() > minLength || word.length() == minLength) {
+                earnedWords.add(word);
+            }
+        }
+        return earnedWords;
+    }
+
+    private static void seventhExercise(Scanner scanner) {
+        ArrayList<String> inputWords = new ArrayList<>();
+        inputWords.add("Qwerty");
+        inputWords.add("Uiop");
+        inputWords.add("Asdfg");
+        inputWords.add("Hjkl");
+        inputWords.add("Zxcvb");
+        System.out.println("Input word: ");
+        String inputWord = scanner.nextLine();
+        System.out.println(findValue(inputWords, inputWord));
+    }
+
+    public static int findValue(ArrayList<String> words, String inputWord) {
+
+        if (words.contains(inputWord)) {
+            return words.indexOf(inputWord);
+        }
+        return -1;
+    }
+
+    private static void eighthExercise() {
+        ArrayList<String> inputWords = new ArrayList<>();
+        inputWords.add("Qwerty");
+        inputWords.add("Qwerty");
+        inputWords.add("Uiop");
+        inputWords.add("Uiop");
+        inputWords.add("Asdfg");
+        inputWords.add("Asdfg");
+        inputWords.add("Hjkl");
+        inputWords.add("Zxcvb");
+        System.out.println(removeDuplicates(inputWords));
+    }
+
+    public static ArrayList<String> removeDuplicates(ArrayList<String> words) {
+        ArrayList<String> earnedWords = new ArrayList<>();
+        for (String word : words) {
+            if (!earnedWords.contains(word) && words.contains(word)) {
+                earnedWords.add(word);
+            }
+        }
+        return earnedWords;
+    }
+
+    private static void twelfthExercise() {
+        ArrayList<Integer> firstList = new ArrayList<>();
+        firstList.add(1);
+        firstList.add(2);
+        firstList.add(3);
+        firstList.add(4);
+        ArrayList<Integer> secondList = new ArrayList<>();
+        secondList.add(1);
+        secondList.add(4);
+        secondList.add(5);
+        secondList.add(9);
+
+        ArrayList<Integer> earnedList = new ArrayList<>();
+
+        findDistinctNumbers(firstList, secondList, earnedList);
+        findDistinctNumbers(secondList, firstList, earnedList);
+
+        System.out.println("Output");
+        System.out.println(earnedList);
+    }
+
+    private static void findDistinctNumbers(ArrayList<Integer> firstList, ArrayList<Integer> secondList, ArrayList<Integer> earnedList) {
```

Comment:
let's make this method returns a result list instead of be void.

Reply:
fixed